pandas/tests/dtypes/test_inference.py

```diff
@@ -1184,6 +1185,10 @@ def test_is_scalar_pandas_containers(self):
         assert not is_scalar(Index([]))
         assert not is_scalar(Index([1]))
 
+    def tes_is_scalar_pep_3141(self):
```

Comment:
just add onto the  ``test_is_scalar_builtin_scalars`` test

Reply:
I tried it, @jreback . Strangely, it was throwing errors, and local tests were failing. That's why I defined it in a different function. I couldn't understand the error either, let me send you a snip.